doc-content/enterprise-only/openshift/ba-openshift-overview-con.adoc

```diff
@@ -1,52 +1,77 @@
+For instructions about deploying this environment, see {URL_DEPLOYING_AUTHORING_MANAGED_ON_OPENSHIFT}[_{DEPLOYING_AUTHORING_MANAGED_ON_OPENSHIFT}_].
+endif::[]
+endif::DM[]
+* _Deployment with immutable servers_: An alternate environment for running existing {ASSETS} for staging and production purposes. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a {ASSET} or group of {ASSETS}. You cannot stop any {ASSET} on the pod or add any new {ASSET} to the pod. If you want to use another version of a {ASSET} or modify the configuration in any other way, you deploy a new server image and displace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment; you can use any container-based integration workflows and do not need to use any other tools to manage the pods. 
+ifdef::PAM[]
+Optionally, you can use {CENTRAL} Monitoring to monitor the performance of the environment and to stop and restart some of the {ASSET} instances, but not to deploy additional {ASSETS} to any {KIE_SERVER} or undeploy any existing ones (you can not add or remove containers). 
+endif::PAM[]
 ifeval::["{context}"!="openshift-ansible-playbook"]
 For instructions about deploying this environment, see {URL_DEPLOYING_IMMUTABLE_ON_OPENSHIFT}[_{DEPLOYING_IMMUTABLE_ON_OPENSHIFT}_].
 endif::[]
 
+You can also deploy a _trial_ or evaluation environment. This environment includes {CENTRAL} and a {KIE_SERVER}. You can set it up quickly and use it to evaluate or demonstrate developing and running assets. However, the environment does not use any persistent storage, and any work you do in the environment is not saved.
+ifeval::["{context}"!="openshift-ansible-playbook"]
+For instructions about deploying this environment, see {URL_DEPLOYING_TRIAL_ON_OPENSHIFT}[_{DEPLOYING_TRIAL_ON_OPENSHIFT}_].
+endif::[]
+
 ifeval::["{context}"!="openshift-ansible-playbook"]
-To deploy a {PRODUCT} environment on OpenShift, you can use the templates that are provided with {PRODUCT}. You can modify the templates to ensure that your environment suits your needs.
+To deploy a {PRODUCT} environment on OpenShift, you can use the templates that are provided with {PRODUCT}. 
+ifdef::PAM[You can modify the templates to ensure that the configuration suits your environment.]
 endif::[]
 ifeval::["{context}"=="openshift-ansible-playbook"]
-You can use the Automation Broker with the {PRODUCT} Ansible Playbook to deploy a {PRODUCT} environment on OpenShift in an interactive process. You can set all possible configuration values during this process. The installation process can generate all the required secrets automatically.
+You can use the Automation Broker with the {PRODUCT} Ansible Playbook to deploy a {PRODUCT} environment on OpenShift in an interactive procedure. You can set all possible configuration values during this procedure. During the installation, the Automation Broker can generate all the required secrets automatically.
```

Comment:
maybe update the first "procedure" occurrence to "mode"?

Reply:
Changed as requested.